server/core/src/actors/v1_read.rs

```diff
@@ -744,7 +744,7 @@ impl QueryServerReadV1 {
                     // get the first entry
                     .and_then(|e| {
                         // From the entry, turn it into the value
-                        e.get_ava_iter_sshpubkeys("ssh_publickey")
+                        e.get_ava_iter_sshpubkeys(Attribute::SshUnderscorePublicKey.into())
```

Comment:
Why spell out the Underscore here?

Reply:
because we have both "sshpublickey" and "ssh_publickey" and at one point I broke everything

Reply:
OHH yeah we do, because of ldap. Couldn't we just have the non-unsdorced version exist in LDAP_ATTR only?

Reply:
Nope, 'cause someone made it a virtual Schema attribute here:

https://github.com/yaleman/kanidm/blob/4371640519dda239a5c6c21468569b846105be28/server/lib/src/schema.rs#L1667

Reply:
We could rename it to LdapSshPublicKey or something?

Reply:
Yeah, because it's there to simulate / prevent people creating it later since it's an ldap attr. So maybe we name it Attribute::SshPublicKey and Attribute::LdapSshPublicKey. Doesn't match the text of the attrs, but describes their use.